src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -320,22 +320,21 @@ def load_profile_properties(self, profile_name: str) -> dict:
             profile_name = ".".join(lst)
             profile = self.find_profile(profile_name, self.profiles)
             if profile is not None:
-                props = { **profile.get("properties", {}), **props }
+                props = {**profile.get("properties", {}), **props}
                 secure_fields.extend(profile.get("secure", []))
             else:
                 warnings.warn(
-                        f"Profile {profile_name} not found",
-                        ProfileNotFoundWarning
-                        )
+                    f"Profile {profile_name} not found",
+                    ProfileNotFoundWarning
+                )
             lst.pop()
 
-
         # load secure props only if there are secure fields
         if secure_fields:
             CredentialManager.load_secure_props()
             self.secure_props=CredentialManager.secure_props.get(self.filepath, {})
             # load properties with key as profile.{profile_name}.properties.{*}
-            for (key, value) in self.secure_props.items():
+            for (key, value) in CredentialManager.secure_props.items():
```

Comment:
I think we still want to use `self.secure_props` here, because it is an object containing secure properties for a config file where the keys are JSON paths and the values are secure credentials.

For the `CredentialManager.secure_props` object, the keys are file paths, and the values are an object structured like `self.secure_props`. The regex for matching JSON paths won't work on file paths.